site/layouts/shortcodes/ods-recommendation-alert.html

```diff
@@ -0,0 +1,13 @@
+{{- /*
+Usage: `ods-recommendation-alert`
+*/ -}}
+
+<div class="bd-ods-alert alert alert-info">
```

Comment:
Did you try with the UI kit v5.0 green color since it is more accessible right now ? Imo success is more understandable compared to info color. Any thoughts @julien-deramond ?

Reply:
We noticed the fact that the info color attract more the attention. Let's keep it as-is for now, it shouldn't be too hard to change it later if needed!